aioresponses/core.py

```diff
@@ -351,3 +352,11 @@ def add(self, url: 'Union[URL, str, Pattern]', method: str = hdrs.METH_GET,
             response.raise_for_status()
 
         return response
+
+
+def get_active() -> aioresponses:
+    from aiohttp.client import ClientSession
+    response = getattr(ClientSession._request, "_aioresponse", None)
+    if response is None:
+        raise ValueError("No active aioresponse mock")
+    return response
```

Comment:
What about returning None instead of raising ValueError?
Would avoid having to use try/except.